packages/coordinator/tests/services/translation/response/common.spec.ts

```diff
@@ -368,14 +368,12 @@ describe("translateAgentPerson", () => {
 
   test("Prescription is Secondary Care, PrescriptionType starts with 1", () => {
```

Comment:
We should probably just remove this test now, it's not really doing what it says anymore

Reply:
Good call